sources/Core/Numerics/Quaternion.cs

```diff
@@ -27,6 +28,22 @@ public Quaternion(float x, float y, float z, float w)
             _w = w;
         }
 
+        /// <summary>
+        /// Initialize a new instance of the <see cref="Quaternion"/> struct from a <see cref="ReadOnlySpan{T}"/>
+        /// </summary>
```

Comment:
The repo uses single line tags where possible.
```suggestion
        /// <summary>Initialize a new instance of the <see cref="Quaternion"/> struct from a <see cref="ReadOnlySpan{T}"/></summary>
```

This applies to all documentation comments, not just this one.